app/views/escorts/_actions.html.slim

```diff
@@ -5,5 +5,8 @@ ul.actions
   - elsif escort.completed?
     li
       = link_to_print_in_new_window(escort, styles: 'button button-print')
+  - unless escort.issued?
```

Comment:
Just above this code we have the reverse query `- if escort.issued?`. Can't this be handled by a single condition?